dashboard/src/shared/GrpcClient.ts

```diff
@@ -0,0 +1,22 @@
+import { grpc } from "@improbable-eng/grpc-web";
+import { GrpcWebImpl } from "gen/kubeappsapis/core/plugins/v1alpha1/plugins";
+import * as URL from "./url";
+
+export class GrpcClient {
+  private baseUrl = URL.api.kubeappsapis;
+  private grpcClient!: GrpcWebImpl;
+
+  private GetClientConfig = () => {
+    return {
+      // https://github.com/improbable-eng/grpc-web/blob/master/client/grpc-web/docs/transport.md#specifying-transports.
+      transport: grpc.CrossBrowserHttpTransport({ withCredentials: true }),
```

Comment:
Did you try without the `withCredentials: true`? From the docs:

```
  // send browser cookies along with cross-origin requests (CORS), defaults to `false`.
  withCredentials?: boolean
```

It's not clear to me from the docs there whether when its false, we'd still get credentials sent since the request isn't cross origin, or whether it means when false it won't send any credentials with any requests whether they satisfy CORS or not.

Reply:
Great, thanks.